native_client/ctcdecode/build_archive.py

```diff
@@ -59,14 +68,25 @@ def build_one(file):
         if os.path.exists(outfile):
             return
 
-        cmd = '{cc} -fPIC -c {cflags} {args} {includes} {infile} -o {outfile}'.format(
-            cc=compiler,
-            cflags=cflags,
-            args=' '.join(args),
-            includes=' '.join('-I' + i for i in INCLUDES),
-            infile=file,
-            outfile=outfile,
-        )
+        if sys.platform.startswith('win'):
```

Comment:
This looks very very much the same. I would suggest avoiding a big `if` there and just make `format()` arguments deal with this.

Reply:
I removed duplicated codes using `format()` arguments.